### PR TITLE
[Intel MKL] Fix dequantize accuracy issue and re-enable this OP

### PR DESCRIPTION
**Pull request by guizili0**

### CODE REVIEW

File: tensorflow/core/kernels/mkl_dequantize_op_test.cc [hunk 1]
```diff
@@ -59,4 +61,83 @@ TEST_F(MklDequantizeOpTest, small) {
...
+                     .Attr("_kernel", "MklLayoutDependentOp")
+                     .Finalize(node_def()));
+    TF_ASSERT_OK(InitOp());
+    AddInputFromArray<T>(tensor.shape(), tensor.flat<T>());
+    AddInputFromArray<uint8>(mkl_meta_tensor.shape(),
+                             mkl_meta_tensor.flat<uint8>());
+    TF_ASSERT_OK(RunOpKernel());
+
+    *output = *GetOutput(0);
+  }
```
Comment (penpornk) [inline tensorflow/core/kernels/mkl_dequantize_op_test.cc]: I remember seeing this in many test files. Can we refactor this to a common test utility file in a future PR? Thank you!

**Review by penpornk — DISMISSED** [review]
Thank you for the PR!

**Comment by penpornk** [comment]
Could you please resolve the conflicts?

File: tensorflow/core/kernels/mkl_dequantize_op_test.cc (hunk 1 above)
Comment (guizili0) [reply tensorflow/core/kernels/mkl_dequantize_op_test.cc]: @penpornk sure, we will move this general code to a util file in the feature.

**Comment by guizili0** [comment]
@penpornk Thanks for the quick review. The conflict has been resolved, please help to check. Thank you!

**Review by penpornk — DISMISSED** [review]
Thank you!

**Comment by penpornk** [comment]
It seems we have a conflict again. Could you please resolve it? Thank you!
We also get a `mkl_dequantize_test` failure in `MacOS CPU Python3` and `MacOS Python2 and CC`. [Link](https://source.cloud.google.com/results/invocations/30c8254d-20bd-4a5e-92ab-101a8fafbb2f/targets/%2F%2Ftensorflow%2Fcore%2Fkernels:mkl_dequantize_op_test/log). But this might be because of the merge conflict. I'll rerun the tests after the conflict is resolved.

**Comment by guizili0** [comment]
@penpornk Thanks for the remind, the conflict has been resolved, please help to check. Thank you!

**Review by penpornk — APPROVED** [review]
Thank you very much!

**Comment by smit-hinsu** [comment]
tensorflow/core/kernels:mkl_dequantize_op_test seems to be failing and this PR seems to be a related change.

Please take a look.

**Comment by guizili0** [comment]
@smit-hinsu Thanks for the reminder, I will check.

**Comment by penpornk** [comment]
@guizili0 Have you had a chance to look at this yet? Would it be possible to get it fixed before the 2.2 branch cut this Wednesday? Currently the tests are disabled in commit https://github.com/tensorflow/tensorflow/commit/ebf01547f5f76ae0c65e708c09d60aa8e06c30a9.

**Comment by guizili0** [comment]
@penpornk Sorry for this late fix. The fix PR is here https://github.com/tensorflow/tensorflow/pull/37030 please help to check. Thank you!

**Comment by penpornk** [comment]
@guizili0 Just approved it. Thank you very much!

**Pull request merged**